packages/next/src/schematics/application/files/tsconfig.json__tmpl__

```diff
@@ -12,5 +12,12 @@
     "resolveJsonModule": true,
     "isolatedModules": true
   },
-  "include": ["**/*.ts", "**/*.tsx", "next-env.d.ts"]
+  "exclude": ["node_modules"],
+  "include": [],
```

Comment:
How come we the includes are empty? Should we be creating a `tsconfig.app.json` file if we're going with solution-style config?